core/http/src/content_type.rs

```diff
@@ -281,11 +281,11 @@ impl From<Mime> for ContentType {
     #[inline]
     fn from(mime: Mime) -> ContentType {
         // soooo inefficient.
-        let params = mime.2.into_iter()
```

Comment:
I had dropped `mime` support entirely.

Reply:
From your comment above, I think you want to keep `mime` support, correct?

Reply:
Yes, if there's no downside, at least. I think it would just add non-async-related churn to an async development branch for now.